src/plugins/wb-disable/disable.js

```diff
@@ -30,15 +30,29 @@ var componentName = "wb-disable",
 			$html = wb.html,
 			i18n = wb.i18n,
 			pageUrl = wb.pageUrlParts,
-			li, param;
+			li, param,
+			i18nTemporary = {
+				en: {
+					"disable-notice-h": "Notice: Basic HTML",
+					"disable-notice": "You are viewing Basic HTML view. Some features may be disabled."
+				},
+				fr: {
+					"disable-notice-h": "Avis: Version HTML simplifiée",
+					"disable-notice": "Vous naviguez présentement sur la version HTML simplifiée de cette page. Certaine fonctionalités peuvent être déactivées."
+				}
+			},
+			noticeHeader = i18n( "disable-notice-h" ),
+			noticeBody = i18n( "disable-notice" ),
```

Comment:
Your suggestion would prevent to show the notification and the possibility for the user to return back in standard mode by clicking a link for languages that are not defined in the i18nTemporary. For example if  ```wb.lang === "it"```